src/main/java/org/eclipse/cargotracker/interfaces/booking/sse/RealtimeCargoTrackingService.java

```diff
@@ -29,19 +29,23 @@ public class RealtimeCargoTrackingService {
   @Inject private CargoRepository cargoRepository;
 
   @Context private Sse sse;
+
   private SseBroadcaster broadcaster;
 
   @PostConstruct
   public void init() {
```

Comment:
I this method is empty, please remove it.

Reply:
Got it, the init() method should be removed in the newest commit.